whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -95,7 +95,10 @@ class RdapObjectMapper {
     private static final String TERMS_AND_CONDITIONS = "http://www.ripe.net/data-tools/support/documentation/terms";
     private static final Link COPYRIGHT_LINK = new Link(TERMS_AND_CONDITIONS, "copyright", TERMS_AND_CONDITIONS, null, null);
 
-    private static final List<String> RDAP_CONFORMANCE_LEVEL = Lists.newArrayList("rdap_level_0");
+    private static final List<String> COMMON_RDAP_CONFORMANCE =
+            Lists.newArrayList(RdapConformance.PROFILE_0.getValue(), RdapConformance.CIDR_0.getValue(),
+            RdapConformance.LEVEL_0.getValue(),
+            RdapConformance.FLAT_MODEL.getValue());
 
```

Comment:
nro_rdap_profile_asn_flat_0 is only for Autnum objects. It should not be added in commons Can you confirm? I do not see it mentioned in section 4.1

Reply:
We are not adding flat in common, we are adding it in /help and in autnums. You are right, we should add it just in autnums. This is why I need a set in /help, because it is calling commons but I can try to refactor this and use a set + add

Reply:
Ahh, I got confuse with the name of constant variable COMMON_RDAP_CONFORMANCE. Can you rename to HELP_RDAP_CONFORMANCE?

Reply:
I will delete it, this is not used now in my branch I will directly read all the values from the Enum to have everything in one place

Reply:
Since mapCommons is already adding the common rdapConformance may be just add the remaining conformances required for help section?

Reply:
I updated the PR right now

Reply:
In that case if tomorrow we need to add a new profile, we will need to add in this list and in the enum as well. To me it makes not sense, I prefer just to add it in the enum and /help will read all of the values that we have in the enum

Reply:
Discussed over chat for refactoring